composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": "^8.2",
-        "illuminate/database": "^11",
-        "illuminate/filesystem": "^11",
-        "illuminate/http": "^11",
-        "illuminate/support": "^11",
-        "illuminate/view": "^11"
+        "illuminate/database": "^12",
+        "illuminate/filesystem": "^12",
+        "illuminate/http": "^12",
+        "illuminate/support": "^12",
+        "illuminate/view": "^12"
     },
     "require-dev": {
```

Comment:
`orchestral/testbench` needs to be upgraded to `^10` support laravel 12

https://github.com/orchestral/testbench/releases/tag/v10.0.0